querybook/webapp/components/AIAssistant/AutoFixButton.tsx

```diff
@@ -35,37 +37,53 @@ export const AutoFixButton = ({
         fixed_query: fixedQuery,
     } = streamData;
 
-    const bottomDOM = fixedQuery && (
-        <div className="right-align mb16">
-            <Button
-                title="Reject"
-                color="cancel"
-                onClick={() => {
-                    setShow(false);
-                }}
-            />
-            <Button
-                title="Apply"
-                color="confirm"
-                onClick={() => {
-                    onUpdateQuery?.(fixedQuery);
-                    trackClick({
-                        component: ComponentType.AI_ASSISTANT,
-                        element: ElementType.QUERY_ERROR_AUTO_FIX_APPLY_BUTTON,
-                    });
-                    setShow(false);
-                }}
-            />
-        </div>
-    );
+    const bottomDOM =
+        streamStatus === StreamStatus.STREAMING ? (
+            <div className="right-align mb16">
+                <Button
+                    title="Stop Generating"
+                    color="light"
+                    onClick={cancelStream}
+                    className="mr8"
+                />
+            </div>
+        ) : (
+            fixedQuery && (
```

Comment:
replace with `streamStatus === StreamStatus.DONE &&` ?

Reply:
it doesn't always have the fixedQuery when the streaming is done. for those cases, we dont want to show the apply/cancel buttons